File: backend/check_db.js
const { Client } = require('pg');

const client = new Client({
  connectionString: 'postgres://neondb_owner:npg_oJ21CsYWzTie@ep-tiny-night-b1ydnyk0.c-5.eu-central-1.aws.neon.tech/neondb?sslmode=require'
});

async function run() {
  await client.connect();
  
  const res = await client.query('SELECT status, COUNT(*) FROM manifest_import_batches GROUP BY status');
  console.log("Batches by status:");
  console.log(res.rows);

  const res2 = await client.query(`
    SELECT b.status, COUNT(m.id) 
    FROM manifest_passengers m 
    JOIN manifest_import_batches b ON m.batch_id = b.id 
    GROUP BY b.status
  `);
  console.log("Passengers by batch status:");
  console.log(res2.rows);

  await client.end();
}

run().catch(console.error);

File: backend/CheckDb.java
import java.sql.*;

public class CheckDb {
    public static void main(String[] args) throws Exception {
        String url = "jdbc:postgresql://ep-tiny-night-b1ydnyk0.c-5.eu-central-1.aws.neon.tech/neondb?sslmode=require&channel_binding=require";
        String user = "neondb_owner";
        String password = "npg_oJ21CsYWzTie";
        
        try (Connection conn = DriverManager.getConnection(url, user, password);
             Statement stmt = conn.createStatement();
             ResultSet rs = stmt.executeQuery("SELECT version, description, type, script, checksum, installed_on, success FROM flyway_schema_history ORDER BY installed_rank DESC LIMIT 5")) {
             
            while (rs.next()) {
                System.out.println("V" + rs.getString("version") + " | " + 
                                   rs.getString("description") + " | " + 
                                   rs.getString("script") + " | " + 
                                   "success: " + rs.getBoolean("success"));
            }
        }
    }
}
